### PR TITLE
Adds the basic console UI for app

### PR DESCRIPTION
**Pull request by HabibPayenda**
### **The following requirements were added to the project:**

- [ ] Create a basic Ui for the app which has the following operations:
- '1 - List all books'
- '2 - List all people'
-  '3 - Create a person'
- '4 - Create a book'
-  '5 - Create a rental'
-  '6 - List all rentals for a given person id'
-  '7 - Exit'

Thank you.

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,181 @@
+require 'colorize'
+require_relative 'student'
```
Comment (thecodechaser) [inline app.rb]: - [ ] When I run the app, I get the below error. If I comment the line `require 'colorize'` the app works fine. Please fix this issue.

image

File: app.rb [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+    puts 'Enter a corresponding number to chose an option:'
+    puts '1 - List all books'
+    puts '2 - List all people'
+    puts '3 - Create a person'
+    puts '4 - Create a book'
+    puts '5 - Create a rental'
+    puts '6 - List all rentals for a given person id'
+    puts '7 - Exit'
+  end
+
```
Comment (thecodechaser) [inline app.rb]: - [ ] Please make sure to add a `main.rb` file that will be the entry point of your app and place this list of options in the file and call the methods from the `app.rb` file based on user selection.

image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @HabibPayenda👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.rb (hunk 2 above)
Comment (HabibPayenda) [reply app.rb]: @thecodechaser Thank you so much, I will implement the required changes in the project.

**Review by Deepakdanger — APPROVED** [review]
Hi @HabibPayenda, 

### Status: Approved ✔️

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚢 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**